Add the notification channels tutorial (#5464)
kyma-project/kyma: resources/monitoring/charts/alertmanager/README.md

# Alertmanager

## Overview 

> This chart contains a part of the configuration related to the Alertmanager.

Reviewer: ```suggestion
This chart contains configuration related to Alertmanager.
```

Author: Done.

> For details on Alertmanager configuration, see [this](https://github.com/kyma-project/kyma/blob/master/docs/monitoring/03-01-alertmanager.md) document.

Reviewer: same as above? full link to the website?

> For details on Alertmanager configuration, see [this](https://github.com/kyma-project/kyma/docs/monitoring/03-01-alertmanager.md) document.

Reviewer: ```suggestion
For details on Alertmanager configuration, see [this](https://github.com/kyma-project/kyma/blob/master/docs/monitoring/03-01-alertmanager.md) document.
```

Reviewer: but I think we should point to the website, hm?

Author: We can but will it work in the `resources` dir?

## Details

> Kyma comes with a set of alerting rules provided out of the box. You can find them [here](https://github.com/kyma-project/kyma/tree/master/resources/monitoring/charts/alertmanager/templates).

Reviewer: ```suggestion
Kyma comes with a set of out-of-the-box alerting rules. You can find them [here](https://github.com/kyma-project/kyma/tree/master/resources/monitoring/charts/alertmanager/templates).
```

Author: Done.

> These rules provide alerting configuration for logging, webapps, rest services, and custom Kyma rules. 

Reviewer: ```suggestion
These rules provide alerting configuration for logging, web applications, REST services, and custom Kyma rules. 
```

Author: Done.

## Alertmanager configuration - alertmanager.yaml

This section explains how to configure Alertmanager to enable alerting notifications. [This](templates/alertmanager.config.yaml) template pre-configures two simple receivers to handle alerts in VictorOps and Slack.

This yaml file pre-configures two simple receivers to handle alerts in VictorOps and Slack.

> To avoid confusion, use optional configuration parameters for `route:` and then group the receivers under the label ```routes:```

Reviewer: ```suggestion
To avoid confusion, use optional configuration parameters for `route:` and then group the receivers under the ```routes:``` label.
```

Author: Done.
